client.go

```diff
@@ -120,10 +120,9 @@ type Client interface {
 	// as a result of the read
 	Read(ctx context.Context, fieldsToRead []string, objectToRead DomainObject) error
 
-	// TODO: Coming in v2.1
 	// MultiRead fetches several rows by primary key. A list of fields can be
 	// specified. Use All() or nil for all fields.
-	// MultiRead(context.Context, []string, ...DomainObject) (MultiResult, error)
+	MultiRead(context.Context, []string, ...DomainObject) (MultiResult, error)
```

Comment:
Can you add comments on how this API supports multiread, and what it does not support? For example, same entity, cross-entity, or cross-scope?